src/core/CCList.mli

```diff
@@ -703,51 +736,51 @@ val random_non_empty : 'a random_gen -> 'a t random_gen
 val random_len : int -> 'a random_gen -> 'a t random_gen
 
 val random_choose : 'a t -> 'a random_gen
-(** Randomly choose an element in the list.
+(** [random_choose l] randomly chooses an element in the list [l].
     @raise Not_found if the list is empty. *)
 
 val random_sequence : 'a random_gen t -> 'a t random_gen
 
 val to_string : ?start:string -> ?stop:string -> ?sep:string ->
   ('a -> string) -> 'a t -> string
-(** [to_string ~start ~stop ~sep item_to_string l] print [l] to a string using
+(** [to_string ~start ~stop ~sep item_to_string l] prints [l] to a string using
     [sep] as a separator between elements of [l].
     @since 2.7 *)
 
 val to_iter : 'a t -> 'a iter
-(** Return a [iter] of the elements of the list.
+(** [to_iter l] returns a [iter] of the elements of the list [l].
     @since 2.8 *)
 
 val to_std_seq : 'a t -> 'a Seq.t
-(** Return a [Seq.t] of the elements of the list.
+(** [to_std_seq l] returns a [Seq.t] of the elements of the list [l].
     @since 2.8 *)
 
 val of_iter : 'a iter -> 'a t
-(** Build a list from a given [iter].
+(** [of_iter iter] builds a list from a given [iter].
     In the result, elements appear in the same order as they did in the source [iter].
     @since 2.8 *)
 
 val of_std_seq_rev : 'a Seq.t -> 'a t
-(** Build a list from a given [Seq.t], in reverse order.
+(** [of_std_seq_rev Seq.t] builds a list from a given [Seq.t], in reverse order.
     @since 2.8 *)
 
 val of_std_seq : 'a Seq.t -> 'a t
-(** Build a list from a given [Seq.t].
-    In the result, elements appear in the same order as they did in the source [seq].
+(** [of_std_seq Seq.t] builds a list from a given [Seq.t].
```

Comment:
`Seq.t` is a strange name for an argument.

Reply:
Seq.t is used in other modules. (CCArray, CCString ...).
For these names (and for now) I suggest keeping them as is. At the end of 'Phase One' 
(this reformatting of comments ), we can establish a complete index of the names and modify them eventually.

Reply:
OK, then can you remove the `Seq.t` from `of_std_seq Seq.t` which is not correct OCaml?

Reply:
must be a search and replace that got out of hand :)

Reply:
@FardaleM  _...which is not correct OCaml_

Sorry, but I don't understand this remark.
- We talk about _Seq.t_ used in COMMENTS
- And even in OCaml code, maybe _Seq.t_ is strange but completely correct.
Tons of programs use it: OCaml stdlib, alt-ergo, lwt, bap, fmt,coccinelle and so on.
 
Anyway, to sum up the situation: 
if I use **seq** instead of **Seq.t** is it OK for you?

Reply:
Yes it is fine for me.
To explain my remark, `[]` is often used to print code in the doc and `of_std_seq Seq.t` looks very strange but I made a mistake, it can be correct in OCaml.

Reply:
You can change it or leave it as you prefer. If it is in other modules as well we need to change all of them at some point.